### DISCUSSION TITLE
Intent to play music in current area

### DISCUSSION THREAD

**Discussion by jonathanarchergithub** [music-assistant/support#3325] [Feature requests and ideas]
Hi, would love to have an intent for MA voice control that allows to play a query in the current area, i.e. a short "Play Nirvana" playing in the area the voice satellite is located in.

**Comment by OzGav** [comment]
This is already the case https://music-assistant.io/integration/voice/

**Reply by jonathanarchergithub** [reply]
Found that the context aware sentences were missing in the file for the MA specific conversation agent. I copied it from the standard assist file and it works. Thanks for the hint.
